### PR TITLE
Upgrade Next.js to 9.5.1

### PR DESCRIPTION
**Pull request by flybayer**
### What are the changes and their implications?

Upgrade Next.js to 9.5.1:
- Release notes: [9.5.0](https://github.com/vercel/next.js/releases/tag/v9.5.0)
- Release notes: [9.5.1](https://github.com/vercel/next.js/releases/tag/v9.5.1)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -11,20 +11,6 @@ on:
       - canary
 
 jobs:
-  check_install_time:
```
Comment (Skn0tt) [inline .github/workflows/main.yml:14]: Shouldn‘t this go into its own PR? 😇

Comment (flybayer) [reply .github/workflows/main.yml:14]: It is failing the build, so just getting rid of it :)
